src/SDKs/Azure.Core/data-plane/Azure.Core/Pipeline/Policies/FixedRetryPolicy.cs

```diff
@@ -15,12 +15,18 @@ public class FixedRetryPolicy : RetryPolicy
 
         protected override void GetDelay(HttpPipelineMessage message, int attempted, out TimeSpan delay)
         {
+            base.GetDelay(message, attempted, out delay);
```

Comment:
Wouldn't we want to wait at least `Delay` time?   If the calculated delay from retry headers is greater, use that.... otherwise, fall back to our stock `Delay`?